modules/network-observability-packet-drops.adoc

```diff
@@ -47,5 +47,5 @@ spec:
 * When you refresh the *Network Traffic* page, the *Overview*, *Traffic Flow*, and *Topology* views display new information about packet drops:
 .. Select new choices in *Manage panels* to choose which graphical visualizations of packet drops to display in the *Overview*.
 .. Select new choices in *Manage columns* to choose which packet drop information to display in the *Traffic flows* table.  
-... In the *Traffic Flows* view, you can also expand the side panel to view more information about packet drops. 
+... In the *Traffic Flows* view, you can also expand the side panel to view more information about packet drops. If any flows are dropped, they are shown in the side panel of the Traffic flow table. Host drops are prefixed with `SKB_DROP` and OVS drops are prefixed with `OVS_DROP`.
```

Comment:
The line starting with `if any`  is not necessary imo. Just the last line is sufficient